notebooks/220-cross-lingual-books-alignment/220-cross-lingual-books-alignment.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    anna_karenina_en[:1500]
</pre><p>can we pretify output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Show a prettified output for the En version too now.